lib/assets/chart_cell/main.css

```
@@ -13,17 +13,20 @@
   --gray-800: #1c2a3a;
 
   --blue-100: #ecf0ff;
+  --blue-600: #3e64ff;
+  --blue-700: #2d4cdb;
 }
 
 .container {
   border: solid 1px var(--gray-300);
   border-bottom: solid 1px var(--gray-200);
-  border-radius: 0.5rem 0.5rem 0 0;
+  border-radius: 0.5rem;
   background-color: rgba(248, 250, 252, 0.3);
```

Add spacing below the add layer button:

```suggestion
  background-color: rgba(248, 250, 252, 0.3);
  padding-bottom: 12px;
```